pkgs/development/python-modules/ansible-pylibssh/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, libssh
+, cython
+, setuptools
+, toml
+, expandvars
+}:
+
+buildPythonPackage rec {
+  pname = "ansible-pylibssh";
+  version = "1.1.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-spaGux6dYvtUtpOdU6oN7SEn8IgBof2NpQSPvr+Zplg=";
+  };
+
+  propagatedBuildInputs = [
+    libssh
+    cython
+    setuptools
+    toml
+    expandvars
+  ];
```

Comment:
Yes, that is correct. `libssh` is a runtime dependency, so I have kept it here, and moved all other dependencies to `nativeBuildInputs`, including the ones you mentioned (except `setuptools-scm-git-archive` since somebody else already updated `setuptools-scm` to 7.1.0, and `git_archival.txt` has been updated already).